lemmy_api/src/community.rs

```diff
@@ -226,6 +241,19 @@ impl Perform for EditCommunity {
     let icon = diesel_option_overwrite(&data.icon);
     let banner = diesel_option_overwrite(&data.banner);
 
+    // Check to make sure the icon and banners are urls
+    if let Some(Some(icon)) = &icon {
+      if Url::parse(icon).is_err() {
+        return Err(APIError::err("invalid_url").into());
+      }
+    }
+
+    if let Some(Some(banner)) = &banner {
+      if Url::parse(banner).is_err() {
+        return Err(APIError::err("invalid_url").into());
+      }
+    }
```

Comment:
Why dont these use `check_optional_url()`?

Reply:
Ah whoops, could've sworn i grepped for Some(Some, my bad.